deploy/docs/Best_Practices.md

```diff
@@ -47,14 +47,21 @@ When Multiline is On, if the first line matches `Parser_Firstline`, the rest of
 Parser_Firstline multi_line
 Parser_1 optional_parser
 ```
-### Collecting Log Lines Over 16KB
+### Collecting Log Lines Over 16KB (with multiline support)
 Docker daemon has a limit of 16KB/line so if a log line is greater than that, it might be truncated in Sumo.
 To fix this, fluent-bit exposes a parameter:  
 ``` bash
 Docker_Mode  On
 ```
-If enabled, the plugin will recombine split Docker log lines before passing them to any parser. This mode cannot be used at the same time as Multiline.
-Reference: https://docs.fluentbit.io/manual/v/1.3/input/tail#docker_mode
+If enabled, the plugin will recombine split Docker log lines before passing them to any parser.
+
+#### Multiline Support
```

Comment:
So this is under `### Collecting Log Lines Over 16KB` section hence maybe(?) change this to `Multiline Support with Docker` as this only touches docker configuration?

Reply:
I think current name is more descriptive for the users if they look for the solution of the issue but doesn't know the insights of docker/fluent-bit too much